CONTRIBUTING.md

```diff
@@ -252,28 +234,67 @@ examples are run. See the relevant section of the
 
 ### Building the documentation locally
 We recommend that you build and view the documentation website locally, before you push it.
-To do so, first install the requirements for building the documentation:
+To do so, first navigate to `docs/`.
+All subsequent commands should be run from within this directory.
+```sh
+cd docs
+```
+Install the requirements for building the documentation:
 ```sh
-pip install -r docs/requirements.txt
+pip install -r requirements.txt
 ```
 
-Then, from the root of the repository, run:
+Build the documentation:
+
+::::{tab-set}
+:::{tab-item} All platforms
+```sh
+python make_api_index.py && sphinx-build source build
+```
+:::
+
+:::{tab-item} Unix platforms with `make`
 ```sh
-sphinx-build docs/source docs/build
+make html
 ```
+:::
+::::
 
 You can view the local build by opening `docs/build/index.html` in a browser.
```

Comment:
This should now be `docs/build/html/index.html`

Reply:
Oh.. they'll be in either depending on how the docs were generated

Reply:
then maybe move the index path inside the following tabset?